test/unit/component/registry/css-registry.test.ts

```diff
@@ -13,45 +13,97 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
 import { CssRegistry } from '../../../../src/component/registry/css-registry';
 
 describe('manage css classes for BPMN cells', () => {
-  let cssRegistry: CssRegistry;
-  beforeEach(() => {
-    cssRegistry = new CssRegistry();
-  });
+  describe('Add css classes', () => {
+    let cssRegistry: CssRegistry;
+    beforeEach(() => {
+      cssRegistry = new CssRegistry();
+    });
 
-  it('1 class name should be registered, when add it for the first time', () => {
-    const bpmnElementId = 'bpmn-id';
-    const classNames = ['class-name'];
+    it('1 class name should be registered, when add it for the first time', () => {
+      const bpmnElementId = 'bpmn-id';
+      const classNames = ['class-name'];
 
-    cssRegistry.addClassNames(bpmnElementId, classNames);
+      cssRegistry.addClassNames(bpmnElementId, classNames);
 
-    expect(cssRegistry.getClassNames(bpmnElementId)).toEqual(classNames);
-  });
+      expect(cssRegistry.getClassNames(bpmnElementId)).toEqual(classNames);
+    });
 
-  it('2 class names should be registered, when add them for the first time', () => {
-    const bpmnElementId = 'bpmn-id';
-    const classNames = ['class-name-1', 'class-name-2'];
+    it('2 class names should be registered, when add them for the first time', () => {
+      const bpmnElementId = 'bpmn-id';
+      const classNames = ['class-name-1', 'class-name-2'];
 
-    cssRegistry.addClassNames(bpmnElementId, classNames);
+      cssRegistry.addClassNames(bpmnElementId, classNames);
 
-    expect(cssRegistry.getClassNames(bpmnElementId)).toEqual(classNames);
-  });
+      expect(cssRegistry.getClassNames(bpmnElementId)).toEqual(classNames);
+    });
+
+    it('a class name should be registered only once, when add it twice', () => {
+      const bpmnElementId = 'bpmn-id';
+      cssRegistry.addClassNames(bpmnElementId, ['class-name-1', 'class-name-2']);
+
+      cssRegistry.addClassNames(bpmnElementId, ['class-name-3', 'class-name-2', 'class-name-4']);
 
-  it('a class name should be registered only once, when add it twice', () => {
-    const bpmnElementId = 'bpmn-id';
-    cssRegistry.addClassNames(bpmnElementId, ['class-name-1', 'class-name-2']);
+      expect(cssRegistry.getClassNames(bpmnElementId)).toEqual(['class-name-1', 'class-name-2', 'class-name-3', 'class-name-4']);
+    });
 
-    cssRegistry.addClassNames(bpmnElementId, ['class-name-3', 'class-name-2', 'class-name-4']);
+    it('getClassNames should return a empty array, when no class name is registered for the BPMN element', () => {
```

Comment:
It should not be in **'Add css classes'** test suite, but in **'Get css classes'** test suite 😉